root/etc/e-smith/templates/etc/netdata/netdata.conf/10base

```diff
@@ -3,6 +3,10 @@
 
     # the default database size - 1 hour
     history = 3600
+    
+    # some defaults to run netdata with least priority
+    process scheduling policy = idle
+    OOM score = 1000
```

Comment:
Why these lines has been added?

Reply:
See https://github.com/netdata/netdata/commit/55c10760e24d4d6cd7553e253237f7279aa4ea59

Reply:
I can't see nothing about OOM score on the linked commit. :thinking: